Mothercell/Hive.IO/Hive.IO/Building/Zone.cs

```diff
@@ -51,7 +51,7 @@ public class Zone
         /// <summary>
         /// Internal loads.
         /// Values indicate full / maximal value.
-        /// Unit in W/sqm
+        /// UnitText in W/sqm
```

Comment:
why change to UnitText?

Reply:
at least it's only doing it in _comments_...